recipes/dd-handler.rb

```diff
@@ -73,4 +73,5 @@
   arguments [handler_config]
   supports :report => true, :exception => true
   action :nothing
-end.run_action(:enable) if node['datadog']['chef_handler_enable']
+  only_if { node['datadog']['chef_handler_enable'] }
```

Comment:
Nice! Worthwhile adding a spec in https://github.com/DataDog/chef-datadog/blob/master/spec/dd-handler_spec.rb to verify the negative (i.e does not load the handler)?